Networking/Networking/Model/OrderItem.swift

```diff
@@ -77,8 +77,14 @@ public struct OrderItem: Decodable, Equatable, Hashable, GeneratedFakeable, Gene
         }
 
         let quantity = try container.decode(Decimal.self, forKey: .quantity)
-        let decimalPrice = try container.decodeIfPresent(Decimal.self, forKey: .price) ?? Decimal(0)
-        let price = NSDecimalNumber(decimal: decimalPrice)
+
+        /// WC versions lower than `6.3` send the item price as a `number`.
+        /// WC Versions equal or greater than `6.3` send the item price as a `string`.
+        ///
+        let decimalPrice = container.failsafeDecodeIfPresent(targetType: String.self,
+                                                             forKey: .price,
+                                                             alternativeTypes: [.decimal { String(describing: $0) }])
+        let price = NSDecimalNumber(string: decimalPrice)
```

Comment:
This is cherry-picked from https://github.com/woocommerce/woocommerce-ios/pull/6226 to allow for continuing the development of the feature